units/host_monitoring_parent_decommission.go

```diff
@@ -81,7 +81,7 @@ func (j *parentDecommissionJob) Run(ctx context.Context) {
 			continue
 		}
 		if idle {
-			err = h.SetDecommissioned(ctx, evergreen.User, false, "container parent has no healthy containers and there is excess capacity")
+			err = h.SetDecommissioned(ctx, evergreen.User, true, "container parent has no healthy containers and there is excess capacity")
```

Comment:
Same thing as above - it seems like this should only decommission if it doesn't have an assigned task.

Reply:
done